lib/brainstem/concerns/controller_param_management.rb

```diff
@@ -0,0 +1,20 @@
+module Brainstem
+  module Concerns
+    module ControllerParamManagement
+      extend ActiveSupport::Concern
+
+      included do
+        class_attribute :brainstem_plural_model_name, :brainstem_model_name,
+                        instance_accessor: false, instance_reader: false, instance_writer: false
```

Comment:
I thought so too!

But not for Rails 3.  `instance_accessor` was added in Rails 4.